x/dex/migrations/v10_to_v11.go

```diff
@@ -10,7 +10,7 @@ import (
 var DexPrefixes = []string{
 	types.LongBookKey,
 	types.ShortBookKey,
-	types.TriggerBookKey,
+	"TriggerBook-value-",
```

Comment:
are these changes necessary?

Reply:
yeah we want to remove TriggerOrder related stuff since it's been causing confusion to the audit team (and future readers)